wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md

```diff
@@ -0,0 +1,359 @@
+# Featured artist showcase beatmaps
+
+Prior to the announcement of upcoming [featured artists](wiki/Featured_Artists), members of the [Mappers' Guild](/wiki/Mappers_Guild) who have either achieved Tier 1 (or higher) in [Guild rank](/wiki/Mappers_Guild#rewards) or been accepting certain [limited-time mystery quests](wiki/Mappers_Guild#quests) set by the Guild in advance are being invited every so often to create exclusive "showcase beatmaps" using songs and other materials from osu!'s featured artist backlog that are yet to be released. When the time to announce the featured artists in question are due, these beatmaps will be treated to a comprehensive coverage in the corresponding [news posts](https://osu.ppy.sh/home/news) and will occasionally be promoted in the [@osugame Twitter](https://twitter.com/osugame) as well.
+
+This article lists all the showcase beatmaps that have been created by the members of the Mappers' Guild over time.
+
+## 2017
+
+### November 2017
```

Comment:
"2017" is redundant here, consider changing to just "November" (this goes for all others as well)